assignments/callbacks.js

```diff
@@ -27,23 +27,39 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return arr.pop();
```

Comment:
pop will indeed return the last item of an array but it also removes it from that array. we don't want to change the array here, just look at the last item. we can always find the last item in an array using `array[array.length-1]`